api/v1alpha3/types.go

```diff
@@ -63,8 +63,15 @@ type VnetSpec struct {
 	Name string `json:"name"`
 
 	// CidrBlock is the CIDR block to be used when the provider creates a managed virtual network.
+	// TODO should just make this a list as is done in https://docs.microsoft.com/en-us/cli/azure/network/vnet?view=azure-cli-latest#az-network-vnet-create
```

Comment:
tracking in https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/618#issuecomment-657821311